### PR TITLE
topology2: merge multi-core test into nocodec topology

### PR DESCRIPTION
**Pull request by RanderWang**
The multi-core topology was created since multi-core feature had a few bugs at that time so it was for validation test. Now this feature works fine so merge it into nocodec topology then it will be tested by github CI. And we don't need to maintain a dedicated multi-core topology.
Screenshot from 2023-09-21 16-00-45

### CODE REVIEW

File: tools/topology/topology2/cavs-nocodec.conf [hunk 1]
```diff
@@ -453,6 +464,7 @@ IncludeByKey.PASSTHROUGH {
 			{
 				index	6
 				direction	playback
+				core_id		$SSP2_PCM_CORE_ID
```
Comment (ranj063) [inline tools/topology/topology2/cavs-nocodec.conf:467]: @RanderWang the suggested change is fine for the regular nocodec topology. But for the passthrough nocodec topology, I'd suggest to keep all core IDs as 0. @plbossart what do you think?

Comment (RanderWang) [reply tools/topology/topology2/cavs-nocodec.conf]: sure, I can image it. Removed.

**Review by ranj063 — APPROVED** [review]

**Review by lgirdwood — APPROVED** [review]

**Comment by RanderWang** [comment]
rebased, thanks

**Comment by lgirdwood** [comment]
@RanderWang are we still expecting some rough edges ?
```
2023-09-26 03:18:58 UTC [ERROR] Caught kernel log error
===========================>>
[  317.740853] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: FW reported error: 6 - Unknown error while processing the request
[  317.741220] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: ipc error for msg 0x47000000|0x0
[  317.741236] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: error: failed to disable target core: 2 for widget pipeline.6
[  317.741249] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: Failed to free connected widgets
[  317.741276] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: ASoC: error at snd_soc_pcm_component_hw_free on 0000:00:1f.3: -22
```

**Comment by RanderWang** [comment]
> @RanderWang are we still expecting some rough edges ?
> 
> ```
> 2023-09-26 03:18:58 UTC [ERROR] Caught kernel log error
> ===========================>>
> [  317.740853] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: FW reported error: 6 - Unknown error while processing the request
> [  317.741220] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: ipc error for msg 0x47000000|0x0
> [  317.741236] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: error: failed to disable target core: 2 for widget pipeline.6
> [  317.741249] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: Failed to free connected widgets
> [  317.741276] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: ASoC: error at snd_soc_pcm_component_hw_free on 0000:00:1f.3: -22
> ```

We have a [BUG] [v2.7-rc1] IPC timed out for 0x47000000|MOD_SET_DX when suspending on ADLP_RVP_NOCODEC https://github.com/thesofproject/sof/issues/8072. I will check this issue later since it has low priority

**Comment by kv2019i** [comment]
@RanderWang #8072 has low priority only because we haven't seen it in a long time (i.e. observed reproduction rate is low). The cases with this PR also look different. 8072 is an IPC timeout, but here we see IPC error response "FW reported error: 4 - The system or resource is busy".

**Comment by kv2019i** [comment]
ping @RanderWang ?

**Comment by RanderWang** [comment]
@kv2019i We get some issues with cavs platforms and I am debugging it

**Comment by RanderWang** [comment]
> > @kv2019i We get some issues with cavs platforms and I am debugging it
> 
> Does the newest update resolve issues ?

I just rebased the code. It is a fw bug and I am still debugging it

**Comment by RanderWang** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
> > > @kv2019i We get some issues with cavs platforms and I am debugging it
> > 
> > 
> > Does the newest update resolve issues ?
> 
> I just rebased the code. It is a fw bug and I am still debugging it

Does the latest update resolve the issue ?

File: zephyr/lib/pm_runtime.c [hunk 2]
```diff
@@ -21,6 +21,7 @@ DECLARE_SOF_UUID("power", power_uuid, 0x76cc9773, 0x440c, 0x4df9,
 DECLARE_TR_CTX(power_tr, SOF_UUID(power_uuid), LOG_LEVEL_INFO);
 
 #if defined(CONFIG_PM_POLICY_CUSTOM)
+struct pm_state_info default_pm_state = {PM_STATE_ACTIVE, 0, 0};
```
Comment (lyakh) [inline zephyr/lib/pm_runtime.c:24]: `const static`

**Comment by kv2019i** [comment]
Only single system PM fail now in https://sof-ci.01.org/sofpr/PR8240/build14811/devicetest/index.html , looks good to go.

**Pull request merged**